book/src/external_state_transitions.md

```diff
@@ -0,0 +1,39 @@
+# External State Transitions (EIP-4788 & EIP-2935)
+
+Some Ethereum Improvement Proposals (EIPs) require state transitions that occur outside of the EVM execution engine, and thus outside of REVM itself. Notably, these are:
```

Comment:
Would formulate this differently, it is not outside of EVM there as there is a `transact_system_call` function that does this change. But this change to state does not have a tx and it is state transition.